ci/jenkins/jobs/macros.yaml

```diff
@@ -253,41 +253,54 @@
           export GO111MODULE=on
```

Comment:
This is likely to conflict with @lzhecheng's PR which refactors the Jenkins scripts: https://github.com/vmware-tanzu/antrea/pull/1240

Reply:
changed according to zhecheng's PR #1240